src/SmartComponents/FormComponents/DescriptiveCheckbox.js

```diff
@@ -14,16 +15,37 @@ const DescriptiveCheckbox = (props) => {
     description,
     isDanger,
     isGlobal,
-    input: { onChange, ...input },
+    checkedWarning,
+    input: { onChange, checked, ...input },
   } = useFieldApi({
     ...props,
     type: 'checkbox',
   });
   const formOptions = useFormApi();
+
+  const descriptionProp = useMemo(() => {
```

Comment:
Instead od `memo` I'd use component directly, the rendering is calculated directly by React and we don't have to keep this in memory.

Reply:
done!